bricks/template/__brick__/{{project_name.snakeCase()}}/.github/workflows/ios_deploy_staging_to_firebase.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       KEYCHAIN_PASSWORD: ${{#mustacheCase}}secrets.KEYCHAIN_PASSWORD{{/mustacheCase}}
       FIREBASE_CLI_TOKEN: ${{#mustacheCase}}secrets.FIREBASE_CLI_TOKEN{{/mustacheCase}}
       FIREBASE_APP_ID_STAGING: ${{#mustacheCase}}secrets.FIREBASE_IOS_APP_ID_STAGING{{/mustacheCase}}
+      GITHUB_RUN_NUMBER: ${{#mustacheCase}}secrets.GITHUB_RUN_NUMBER{{/mustacheCase}}
```

Comment:
This env should not be only added to Firebase, but all CD workflows that build and deploy the iOS app

![image](https://github.com/nimblehq/flutter-templates/assets/16315358/ea39b5b1-375d-4723-9dcb-43d3a7509351)